### PR TITLE
[16.0][MIG] sale_rental

### PR DESCRIPTION
**Pull request by Murtaza-OSI**

### CODE REVIEW

**Comment by chandni299** [comment]
Hello @Murtaza-OSI , If I am not wrong then we need to migrate the first module `sale_start_end_dates` to 16.0 as it is part of `__manifest__.py` file of module `sale_rental` here

**Comment by pedrobaeza** [comment]
This one should be definitively be moved to OCA/vertical-rental on this version.

File: sale_rental/tests/test_sale_rental.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        }
+
+        # with so_form.order_line.new() as line:
+        #     line.product_id = self.test_rental_prod
+        # line.start_date = "2022-01-01"
+        # line.end_date = "2022-01-10"
+        # line.rental_qty = 1
+        so = so_form.save()
+        so.write({"order_line": [(0, 0, line_vals)]})
+        sol = so.order_line
```
Comment (Kev-Roche) [inline sale_rental/tests/test_sale_rental.py:33]: In order to play the onchange
```suggestion
        with so_form.order_line.new() as line:
            line.product_id = self.test_rental_prod
            line.start_date = "2022-01-01"
            line.end_date = "2022-01-10"
            line.rental_qty = 1
            line.product_uom_qty = 10
        so = so_form.save()
        sol = so.order_line
```

File: sale_rental/tests/test_sale_rental.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        self.assertEqual(sol.price_subtotal, 60)
```
Comment (Kev-Roche) [inline sale_rental/tests/test_sale_rental.py:34]:
```suggestion
        self.assertEqual(sol.price_subtotal, 600)
```

File: sale_rental/tests/test_sale_rental.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        so = so_form.save()
+        line_vals = {
+            "product_id": self.test_rental_prod.rented_product_id.id,
+            "name": "Test",
+            "display_type": False,
+            "product_uom": self.env.ref("uom.product_uom_day").id,
+            "product_uom_qty": 1,
+            "sell_rental_id": rental,
+        }
+        so.write({"order_line": [(0, 0, line_vals)]})
```
Comment (Kev-Roche) [inline sale_rental/tests/test_sale_rental.py:66]: The line values are already defined in the upper lines. 
```suggestion
```

**Review by Kev-Roche — CHANGES_REQUESTED** [review]
Hi @Murtaza-OSI do you plan to fix this PR and move it top OCA/vertical-rental like suggested by @pedrobaeza ? 
If no, I can do it.

**Comment by coffeinisfun** [comment]
I have made a PR to move this module to OCA/vertical-rental like suggested by @pedrobaeza: https://github.com/OCA/vertical-rental/pull/45

**Pull request closed without merging**